todo-app/src/Components/todoItem.js

```diff
@@ -0,0 +1,94 @@
+//jshint esnext: true
+
+import React, { Component, Fragment } from 'react'
+
+import './styles/todoItem.css';
+import Button from './Button';
+import InputField from './InputField';
+
+
+export default class TodoItem extends Component {
+
+    handleRemove = () => { 
+        this.props.handleRemove(this.props.data.id);
+    }
+
+    handleToggleEdit = () => { 
```

Comment:
In my opinion, it makes it very confusing to have both local and props-based functions which are named the same. But this is especially the case here, because of the argument signature of the original `handleToggleEditing` in your `App` component.